missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/Game.java

```diff
@@ -152,40 +149,64 @@ public Game(Lobby lobby) {
         Bukkit.getScheduler().runTaskLater(MissileWars.getInstance(), () -> applyForAllPlayers(this::runTeleportEventForPlayer), 2);
 
         if (Config.isSetup()) {
-            Logger.WARN.log("Did not fully initialize lobby " + lobby.getName() + " as the plugin is in setup mode");
+            Logger.WARN.log("Did not fully initialize lobby \"" + lobby.getName() + "\" as the plugin is in setup mode");
             return;
         }
 
         // choose the game arena
         if (lobby.getMapChooseProcedure() == MapChooseProcedure.FIRST) {
             setArena(lobby.getArenas().get(0));
+            finalGamePreparations();
+
         } else if (lobby.getMapChooseProcedure() == MapChooseProcedure.MAPCYCLE) {
             final int lastMapIndex = cycles.getOrDefault(lobby.getName(), -1);
             List<Arena> arenas = lobby.getArenas();
             int index = lastMapIndex >= arenas.size() - 1 ? 0 : lastMapIndex + 1;
             cycles.put(lobby.getName(), index);
             setArena(arenas.get(index));
+            finalGamePreparations();
+
         } else if (lobby.getMapChooseProcedure() == MapChooseProcedure.MAPVOTING) {
-            if (lobby.getArenas().size() == 1) {
+            if (mapVoting.onlyOneArenaFound()) {
                 setArena(lobby.getArenas().get(0));
+                Logger.WARN.log("Only one arena was found for the lobby \"" + lobby.getName() + "\". The configured map voting was skipped.");
+                finalGamePreparations();
+            } else {
+                mapVoting.startVote();
             }
-            lobby.getArenas().forEach(arena -> votes.put(arena.getName(), 0));
         }
 
         scoreboardManager = new ScoreboardManager(this);
         scoreboardManager.createScoreboard();
+    }
+
+    /**
+     * This method performs the final preparations for the game start.
+     * <p>
+     * It is necessary that the arena - even in the case of a map vote - is
+     * now already defined.
+     */
+    public void finalGamePreparations() {
```

Comment:
Rename to `makeFinalGamePreparations` or `prepareGame`, method names should start with a verb